plugins/chat/spec/fabricators/chat_fabricator.rb

```diff
@@ -68,7 +68,7 @@
 Fabricator(:chat_message_without_service, class_name: "Chat::Message") do
   user
   chat_channel
-  message { Faker::Alphanumeric.alpha(number: SiteSetting.chat_minimum_message_length) }
+  message { Faker::Alphanumeric.alpha(number: [10, SiteSetting.chat_minimum_message_length].max) }
```

Comment:
I changed the `min` to 10 so that we don't generate duplicate chat messages when `chat_minimum_message_length` is 1 (which is the default).